modules/contact/test/tests/bouncing-block-contact_pdass/try_gitterle.normal.i

```diff
@@ -0,0 +1,267 @@
+starting_point = 2e-1
```

Comment:
Oops, it looks like this file isn't tested in anyway

Reply:
Good catch. I always manage to forget removing all the input files I used for testing...